### PR TITLE
docs(readme): update plugin reference

### PR DESCRIPTION
**Pull request by mfridman**
Hey there, I dropped a comment earlier but figured I'd also update the README to point at the updated plugin. The source of this plugin is in this repository:

https://github.com/bufbuild/plugins

And published on [buf.build ](https://buf.build/) on the plugin page: [community/stephenh-ts-proto](https://buf.build/community/stephenh-ts-proto)

### CODE REVIEW

File: README.markdown [hunk 1]
```diff
@@ -110,7 +110,7 @@ You can also use the official plugin published to the Buf Registry.
 ```yaml
 version: v1
 plugins:
-  - remote: buf.build/stephenh/plugins/ts-proto
+  - plugin: buf.build/community/stephenh-ts-proto
```
Comment (mfridman) [inline README.markdown:113]: Newer versions of the `buf` CLI will use the `plugin` key for both local and remote plugins.

**Comment by mfridman** [comment]
I can also remove the old `push_to_buf_registry` job in .github/workflows/build.yml as well as ts-proto.Dockerfile if you'd like.

**Comment by stephenh** [comment]
Ah great! Thanks for the PR @mfridman ! I'm just scanning it while in between things, but really appreciate the update.

> I can also remove the old push_to_buf_registry job in .github/workflows/build.yml as well as ts-proto.Dockerfile if you'd like.

Yeah, that'd be great, thanks!

**Comment by mfridman** [comment]
Updated. 

The nice thing is you don't have to worry about the ceremony around packaging, testing, and deploying the plugin to buf.build. It's all automated from our side for popular community plugins via github.com/bufbuild/plugins.

**Comment by stephenh** [comment]
Erm, shoot... Sorry @mfridman , I was moving too quickly and am now 2nd guessing myself; can we keep the `ts-proto.Dockerfile`? I believe we have some community members who use it in their own setups.

**Comment by mfridman** [comment]
> Erm, shoot... Sorry @mfridman , I was moving too quickly and am now 2nd guessing myself; can we keep the `ts-proto.Dockerfile`? I believe we have some community members who use it in their own setups.

Sure thing, reverted in https://github.com/stephenh/ts-proto/pull/820/commits/0660cd0216e837bd7f3b959eb626a6782532f4e5

**Comment by stephenh** [comment]
Sweet, thanks @mfridman !

**Pull request merged**